### PR TITLE
Bugs/8522

### PR DESCRIPTION
**Pull request by sam-warren**
## Status
**Ready**

## Fixes or Implements [AB#8522](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8522)
- [X] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
Changed SMS to Phone in text field, fix period spacing

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [X] Not Required

### UI Changes
YES

### Browsers Tested
- [X] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -2,3 +2,6 @@
 # Set default behavior to automatically normalize line endings.
 ###############################################################################
 * text=auto
+
+*.vue text eol=lf
```
Comment (FuriousLlama) [inline .gitattributes:6]: not sure this is what we want since I think all files should be in lf. Thoughts?

Comment (sam-warren) [reply .gitattributes:6]: want me to change it to `* text eol=lf`?

Comment (sslaws) [reply .gitattributes:6]: I think we do!

Comment (FuriousLlama) [reply .gitattributes:6]: Uhm not sure. Do we also want the CS and json files to be lf?

**Pull request merged**